pytest_fixtures/broker.py

```diff
@@ -39,3 +39,25 @@ def rhel77_contenthost_class(request):
     with VMBroker(host_classes={'host': ContentHost}, **BROKER_RHEL77) as host:
         request.cls.content_host = host
         yield
+
+
+@pytest.fixture
+def satellite68_snap20():
+    """A fixture that provides a 6.8 Snap 20 Satellite"""
+    with VMBroker(
+        host_classes={'host': Satellite},
+        workflow='deploy-sat-lite',
+        tempalte='tpl-sat-lite-6.8.0-20.0-rhel-7',
```

Comment:
these are currently hard-coded to snap 20, but after some changes in satlab, we should be able to specify GA.

Reply:
typo in template.

I think these should be read from settings, not hard coded in the fixture.